third_party/repositories/scala_2_13.bzl

```diff
@@ -81,6 +81,10 @@ artifacts = {
             "@io_bazel_rules_scala_scala_library",
         ],
     },
+    "org_scalameta_semanticdb_scalac": {
+        "artifact": "org.scalameta:semanticdb-scalac_2.13.6:4.7.3",
```

Comment:
Given that ScalaMeta version is repeated a few times in this file:

```bzl
    "org_scalameta_scalameta": {
        "artifact": "org.scalameta:scalameta_2.13:4.3.24",
....
    "org_scalameta_trees": {
        "artifact": "org.scalameta:trees_2.13:4.3.24",
```

I suggest that it should be factored out as a variable similar to `scala_version` at line 1:

```bzl
scala_version = "2.13.6"
```

and change the above to

```bzl
        "artifact": "org.scalameta:semanticdb-scalac_{}:{}".format(scala_version, scalameta_version),
```

Reply:
Will do in follow up PR. Seems to have a few gotchas, as it also requires ScalaFmt rules to be updated.

Reply:
From my own experience - keeping versions duplicated allows simple grep for artifact coords